components/ifc-viewer.tsx

```
@@ -694,6 +772,195 @@ function ViewerContent() {
+                type: node.type,
+                match: match,
+                hasMesh: allMeshes[model.modelID]?.[node.expressID] !== undefined,
+                textSample: text.substring(0, 100)
+              });
+            }
+          } catch (err) {
+            errorCount++;
+            if (errorCount <= 3) {
+              console.warn(`Error fetching props for ${model.modelID}-${node.expressID}:`, err);
+            }
+            continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)